src/charge_controller/dev/LCDDisplay.cpp

```diff
@@ -18,47 +16,140 @@ void LCDDisplay::setChargeControllerStatus(const char* str) {
 }
 
 void LCDDisplay::display() {
-    lcd.setTextForSection(0, batteryOneStatus);
+    ControllerModel::Page newPage = model.getPage();
+    switch (newPage) {
+    case ControllerModel::Page::MAIN: {
```

Comment:
```suggestion
    case ControllerModel::Page::MAIN:
```
Our standard is to not include additional brackets around each case. It probably doesn't affect the code, but it results in 2 braces at the same indentation level at the end, which is ugly

Reply:
Normally I wouldn't use the brackets, but the compiler complains otherwise. I think it's because we initialize a variable in the case statement, so the brackets define the scope of the variable, and without them there would be a variable that could be technically in scope but never declared.